docs/reference/cluster_manifest.md

```diff
@@ -212,6 +212,8 @@ explanation of `ttl` and `loop_wait` parameters.
   automatically created by Patroni for cluster members and permanent replication
   slots. Optional.
 
+* **synchronous_mode**
+  Patroni `synchronous_mode` parameter value. The default is set to `false`. Optional.
```

Comment:
```suggestion
  Patroni `synchronous_mode` parameter value. Optional.

```

Reply:
There's no default specified in the go code and generally we don't do it for manifest fields. Just say it's optional as we do with other flags e.g. `enableMasterLoadBalancer` (and add a blank line between next header)

Reply:
Does the same but is shorter. So we can customize it if you like.